src/cellranger/processes/count.nf

```diff
@@ -27,6 +27,24 @@ def generateCellRangerCountCommandDefaults = {
 	} else if (processParams.containsKey('chemistry')) {
 		_chemistry = processParams.chemistry
 	}
+
+	_includeIntrons = null
+	if(processParams.containsKey('includeIntrons')) {
+		if (processParams.includeIntrons == true) {
+			_includeIntrons = true
+		} else {
+			_includeIntrons = false
+		}
+	}
+
+	_noBam = null
+	if(processParams.containsKey('noBam')) {
+		if (processParams.noBam == true) {
+			_noBam = true
+		} else {
+			_noBam = false
+		}
+	}
```

Comment:
This can be simplified by:
```
_noBam = processParams?.noBam
```
And actually, you could even consider to remove the use of this local variable

Reply:
I used `== true` because I don't want to allow matches when something other than a boolean value is used, which is also why I used the local variable.

Reply:
In that case shouldn't there be an error triggered if it's not boolean (e.g.: `"true"`) because it could be misleading for the user since no check are performed on the type of variables ?

Reply:
I hadn't considered doing that because I don't think we're doing it elsewhere, but I'm happy to make it this if you'd prefer:
```groovy
if (processParams.noBam == true) {
	_noBam = true
} else if (processParams.noBam == false) {
	_noBam = false
} else {
	throw new Exception('noBam option must be a boolean value.')
}
```

Reply:
Yes true but often typing checks are also done by `argparse` or other package. Here arguments are passed directly to the command. 
So I prefer doing this and I think we should do this for others especially in that case (arguments not parsed by a package)